dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/GridHeader.java

```diff
@@ -101,6 +102,17 @@ public GridHeader( String name, String column )
         this.column = column;
     }
 
+    /**
+     * @param name formal header name.
+     * @param valueType the header value type.
+     */
+    public GridHeader( String name, ValueType valueType )
+    {
+        this( name );
```

Comment:
I think this is bad practice to call a less argument constructor from one with more. It should be the other way around so that the one with less basically just provides defaults for the parameters that are not passed.

Reply:
I think I can agree. One challenge is if you want to have "sensible" defaults for certain properties and you want to avoid spreading those default values out to many constructors. You could use a variable of course. We have the more-calling-less pattern for all constructors already so I think I don't want to go into a refactor in this PR.